python/main/pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[build-system]
+requires = ["maturin>=0.13,<0.14"]
```

Comment:
Is there a specifc reason to add an upper version bound? Generally they [do more harm than good](https://iscinumpy.dev/post/bound-version-constraints/) - I would prefer to keep this open so the project isn’t stuck on an old maturin.

Reply:
No reason, i will remove it.

Reply:
Cheers